frontend/src/components/Card/card-styles.js

```diff
@@ -7,4 +7,18 @@ export const card = (theme) => css`
   height: max-content;
 `;
 
-export default card;
+export const card___clickable = (theme) => css`
+  background-color: ${theme.color.neutral[0]};
+  border-radius: 16px;
+  box-shadow: 0px 12px 24px -12px rgba(0, 25, 68, 0.2);
+  height: max-content;
+
+  transition: background-color 0.3s;
+  cursor: pointer;
+
+  .card___clickable:hover,
+  .card___clickable:focus,
+  .card___clickable:active {
+    background-color: ${theme.color.neutral[0]};
+  }
```

Comment:
ay wait mu gana ra ni pat? ang better diay nga pamaagi kay i replace ang mga `.card___clickable` to `&` ra, so balig ma `&:hover` so on and so forth, my bad